.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description of Changes
+---
+Replace this text with an overview of your changes made in this Pull Request. Please use your best judgement here, do not be verbose to the point that you are giving an exact step-by-step of your workflow, but do not undersell the changes made. If this Pull Request addresses an open issue, you should reference that too.
```

Comment:
I believe you can use xml comments in markdown `<!-- text here -->` this way you could place the hints here but the user would need to replace the text and could just leave it. Not sure though.

Reply:
Yes, you can use the comment syntax there. Will go ahead and revise, thanks for the feedback.